extension/src/pipeline/index.ts

```diff
@@ -107,6 +112,22 @@ export class Pipeline extends DeferredDisposable {
     return this.addPipeline()
   }
 
+  public async addTopLevelPlot() {
+    const cwd = await this.getCwd()
```

Comment:
[Q] What does the workflow look like if there is no `dvc.yaml` available in the project? I.e. I am in "dvclive-only" mode (e.g. [dvclive-exp-tracking](https://github.com/iterative/dvclive-exp-tracking)) and I try to add a plot... what happens? This will add a root level `dvc.yaml` but will that work unless the train script is added as well? Not a blocker but need to check/update if it is broken.

Reply:
It automatically runs the "create a stage" workflow instead. While technically works, we might want to add a toast with an action to run the "create a stage" workflow instead 🤔.